features/aanhef.feature

```diff
@@ -0,0 +1,80 @@
+  1. Het voorvoegsel van de eerste geslachtsnaam in de aanhef wordt met een hoofdletter geschreven.
+  2. Wanneer een naamcomponent geen waarde heeft, wordt ook de spatie erna niet opgenomen.
+
+  Wanneer de persoon een adellijke titel of predikaat heeft, en de persoon gebruikt de eigen naam (aanduidingNaamgebruik ongelijk aan "P"), dan wordt de aanhef gevuld volgens de volgende tabel:
+  | adellijkeTitelOfPredikaat | aanhef                |
+  | Baron                     | Hoogwelgeboren heer   |
+  | Barones                   | Hoogwelgeboren vrouwe |
+  | Graaf                     | Hooggeboren heer      |
+  | Gravin                    | Hooggeboren vrouwe    |
+  | Hertog                    | Hoogwelgeboren heer   |
+  | Hertogin                  | Hoogwelgeboren vrouwe |
+  | Jonkheer                  | Hoogwelgeboren heer   |
+  | Jonkvrouw                 | Hoogwelgeboren vrouwe |
+  | Markies                   | Hoogwelgeboren heer   |
+  | Markiezin                 | Hoogwelgeboren vrouwe |
+  | Prins                     | Hoogheid              |
+  | Prinses                   | Hoogheid              |
+  | Ridder                    | Hoogwelgeboren heer   |
+
+
+  Abstract Scenario: Aanhef
+    Als de kadasternatuurlijkpersoon wordt geraadpleegd
+    Dan heeft de gevonden persoon naam.aanhef="<aanhef>"
+
+    Voorbeelden: persoon heeft geen adellijke titel of predikaat
+      | aanduidingNaamgebruik | geslachtsaanduiding | VV     | GN       | VP     | GP       | aanhef                                      |
+      | E                     | 1 (Man)             | in het | Veld     | van    | Velzen   | Geachte heer In het Veld                    |
+      | E                     | 1 (Man)             |        | Groenen  |        |          | Geachte heer Groenen                        |
+      | N                     | 2 (Vrouw)           | in het | Veld     | van    | Velzen   | Geachte mevrouw In het Veld-van Velzen      |
+      | N                     | 2 (Vrouw)           |        | Groenen  |        | Groenink | Geachte mevrouw Groenen-Groenink            |
+      | P                     | 2 (Vrouw)           | in het | Veld     | van    | Velzen   | Geachte mevrouw Van Velzen                  |
+      | P                     | 2 (Vrouw)           |        | Groenink |        | Groenen  | Geachte mevrouw Groenen                     |
+      | V                     | 3 (Onbekend)        | van    | Velzen   | in het | Veld     | Geachte heer/mevrouw In het Veld-van Velzen |
+
+    Voorbeelden: persoon heeft een adellijke titel of predikaat
+      | aanduidingNaamgebruik | AT/PK | geslachtsaanduiding | VV      | GN          | VP      | GP       | aanhef                       |
+      | E                     | G     | 1 (Man)             | van den | Aedel       | van der | Veen     | Hooggeboren heer             |
```

Comment:
`GF` voor `Graaf`? Toegegeven, gezien het een man is begrijp ik ook wel dat de `G` van `Graaf` en niet `Gravin` is.

Reply:
Dit zijn codes uit de volgende IMKAD waardenlijst:
https://www.kadaster.nl/schemas/waardelijsten/AdellijkeTitelOfPredikaat/

Reply:
titels (waarden) gebruikt i.p.v. codes. Is duidelijker